integration_tests/payloads/fakeBillingReport.csv

```diff
@@ -0,0 +1,40 @@
+Idp Entity Id,Timestamp,Request Id,Session Id,Hashed Persistent Id,Minimum Level Of Assurance,Preferred Level Of Assurance,Provided Level Of Assurance,Billable Status,Previous Billed Event Request Id,Previous Billed Event Timestamp,RP Entity Id,Response type
```

Comment:
should this live in config with the other fake data?

Reply:
Can be moved to config if we required to do, but this file mainly used for integration testing

Reply:
Keep in main repo as discussed.